src/app/components/code-box/code-box.scss

```diff
@@ -0,0 +1,23 @@
+.code-box-header {
+  background-color: silver;
+  padding: 5px 10px;
+  text-transform: uppercase;
+  font-size: 80%;
+  font-weight: bold;
+}
+
+.code-box-run-button {
+  float: right;
```

Comment:
try using flex-box

Reply:
Using flexbox seems to be overkill to avoid floats, so I went with absolute positioning instead.